public/faq.php

```diff
@@ -0,0 +1,297 @@
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-header" id="headingThree">
+            <a class="" data-toggle="collapse" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
+              <h5 class="mb-0 faq-card-heading">
+                <span class="faq-black-dot">&#9679; &nbsp;&nbsp;</span> What is the minimum amount to invest?
+                <span class="faq-getting-started-icon">+</span>
+              </h5>
+            </a>
+          </div>
+          <div id="collapseThree" class="collapse" aria-labelledby="headingThree" data-parent="#accordion">
+            <div class="card-body">
+              Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod.
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-header" id="headingFour">
+            <a class="" data-toggle="collapse" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+              <h5 class="mb-0 faq-card-heading">
+                <span class="faq-black-dot">&#9679; &nbsp;&nbsp;</span> How secured are my savings?
+                <span class="faq-getting-started-icon">+</span>
+              </h5>
+            </a>
+          </div>
+          <div id="collapseFour" class="collapse" aria-labelledby="headingFour" data-parent="#accordion">
+            <div class="card-body">
+              Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod.
+            </div>
+          </div>
+        </div>
+
+        <div class="card">
+          <div class="card-header" id="headingFive">
+            <a class="" data-toggle="collapse" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
+              <h5 class="mb-0 faq-card-heading">
+                <span class="faq-black-dot">&#9679; &nbsp;&nbsp;</span> Can I upgrade my saving plan in the future?
+                <span class="faq-getting-started-icon">+</span>
+              </h5>
+            </a>
+          </div>
+          <div id="collapseFive" class="collapse" aria-labelledby="headingFive" data-parent="#accordion">
+            <div class="card-body">
+              Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod.
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+</section>
+<!-- /. Getting started accordion -->
+
+<!-- Payment accordion -->
+<section class="faq-getting-started" id="payment">
```

Comment:
Even though this might not pose a problem now, adding `getting-started` to the classes here can be very confusing since `Getting Started` is one of the sections. Can you change it to something neutral?